pages/features.html

```diff
@@ -6,104 +6,134 @@
+      {% trans "Compatibile with music files in lossless FLAC, WAV, and AIFF formats as well as lossy MP3, M4A/AAC, Ogg Vorbis, and Opus formats." %}
     </p>
 
     <p><b>{% trans "EQ and Crossfader Control" %}</b><br>
-      {% trans "Mixxx features multiple choices of audiophile filters with adjustable EQ shelves. Crossfader curve control lets you deliver quick cuts or long, smooth crossfades" %}.
+      {% trans "Choose between multiple audiophile filters with adjustable EQ shelves. Crossfader curve control lets you deliver quick cuts or long, smooth crossfades." %}
     </p>
   </div>
-
-  <div style="clear: both;"></div>
-
-  <div>
-    <H2>{% trans "Quad Sampler Decks" %}</H2>
+  
+    <div style="clear: both;"></div>
+      <h2>{% trans "Quad Sampler Decks" %}</h2>
     <p>
       {% trans "Perfect for dropping that vocal sample or airhorn, our quad sampler decks are routable through either playback channel." %}
     </p>
 
     <center><img src="{% static '/static/images/feature_sampler2.png' %}"></center>
-
-    <H2>{% trans "Built-in Effects" %}</H2>
+    
+    <h2>{% trans "Built-in Effects" %}</h2>
     <p>
-      {% trans "Add sizzle to your mix with reverb, echo, flange, and more.  Assign a Quick Effect to your controller's filter knob to add color to your sound instantly.  Bad PA system or lousy room?  Use the built-in master EQ to smooth out the sound." %}
+      {% trans "Use reverb, echo, flange, and more effects to blend music in your own signature ways. Assign a Quick Effect to your controller's filter knob to add color to your sound instantly.  Bad PA system or lousy room?  Use the built-in master EQ to smooth out the sound." %} <!--TODO: Say a little more-->
     </p>
 
-    <div style="clear: both;"></div>
+    <!--TODO: Insert FX screenshot here-->
+  <div style="clear: both;"></div>
+  
+    <h1>{% trans "Customizable DJ Hardware Support" %}</h1>
+    {% trans "Free as in freedom as well as price, Mixxx plays nice with a variety of hardware without artificial restrictions or vendor lock-in mechanisms. The free version is the full version. Pick and choose gear from any source to build your unique setup." %}<!--TODO: Alignment somehow is messed up here.-->
+    
+    <div class="halfbox_left">
+      <div style="height: 200px;">
+        <center><img width="340px" src="{% static '/static/images/feature_midi2.png' %}" style="padding-top: 20px;"></center>
+      </div>
+
+      <h2>{% trans "MIDI and HID Controller Support" %}</h2>
+      <p>
+        {% blocktrans with hardware_compatibility_wiki_link="/wiki/doku.php/hardware_compatibility" %}<b>Included Presets</b><br> for controllers such as the Traktor Kontrol S4 MK2, Pioneer DDJ-SB &amp; DDJ-SB2, Allen &amp; Heath Xone K2, Hercules DJ Console Series, and many more. <a href="{{ hardware_compatibility_wiki_link }}">Full list of supported devices</a>{% endblocktrans %}
```

Comment:
That would imply that all the controllers listed on the wiki are fully supported. As much as that has improved recently, it's not true for a lot of older controllers.